hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyVerifier.java

```diff
@@ -53,26 +49,15 @@ public interface KeyVerifier {
      * Additionally, the VerificationAssistant provided may modify the result for "primitive", "Contract ID", or
      * "Delegatable Contract ID" keys, and will be called to observe and reply for each such key as it is processed.
      *
+     * <p>The signatures of required keys are guaranteed to be verified. Optional signatures may still be in the
+     * process of being verified (and therefore may time out). The timeout can be configured via the configuration
+     * {@code hedera.workflow.verificationTimeoutMS}
+     *
      * @param key the key to get the verification for
      * @param callback a VerificationAssistant callback function that will observe each "primitive", "Contract ID", or
      * "Delegatable Contract ID" key and return a boolean indicating if the given key should be considered valid.
-     * @return the verification for the given key, or {@code null} if no such key was provided during pre-handle
+     * @return the verification for the given key
      */
     @NonNull
     SignatureVerification verificationFor(@NonNull Key key, @NonNull VerificationAssistant callback);
-
-    /**
-     * Look for a {@link SignatureVerification} that applies to the given hollow account.
-     * @param evmAlias The evm alias to lookup verification for.
-     * @return The {@link SignatureVerification} for the given hollow account.
-     */
-    @NonNull
```

Comment:
Why should these be removed from here and add to `AppKeyVerifier` ? I must be missing something

Reply:
This functionality is not used by any of the services and therefore not needed in the SPI (unless we plan to use it soon). 

`AppKeyVerifier` defines additional functionality that is only used internally by `hedera-app`. An interface is necessary because there are at least two implementations.